schema/deploy/table_organisation_facility.sql

```diff
@@ -0,0 +1,27 @@
+-- Deploy ggircs-portal:table_organisation_facility to pg
+-- requires: schema_ggircs_portal
+-- requires: table_organisation
+-- requires: table_facility
+
+
+begin;
+
+  create table ggircs_portal.organisation_facility (
```

Comment:
Do we need this table, or should we have an `organization_id` foreign key in the `facility` table instead?